pom.xml

```diff
@@ -79,6 +79,7 @@
         <!-- Cannot update to AWS SDK 1.11+ because of Jackson incompatibility.
         Need to update Druid to use Jackson 2.6+ -->
         <aws.sdk.version>1.10.77</aws.sdk.version>
+	      <druid.null.handling.useDefaultValueForNull>true</druid.null.handling.useDefaultValueForNull>
```

Comment:
how about set the default to false and test the null supported path by default since that is going to be the default case anyway.

Reply:
Keep after all library versions